examples/apache_libcloud/public-cloud-modular_instance.py

```diff
@@ -0,0 +1,194 @@
+"""Script used to import not monitored servers from cloud into Cyberwatch and delete terminated cloud servers"""
+
+# Based on the Apache Libcloud API https://libcloud.apache.org/index.html
+# Tested on Google Compute Engine and AWS EC2
+
+# Prerequisites :
+# - install libcloud with command "pip3 install apache-libcloud"
+# faire prerewque vendor neutral
+# - setup cloud to get an API key with https://libcloud.readthedocs.io/en/stable/compute/drivers/
+# - SSH key file of servers to import named "id_rsa"
+# - If you are not using the default credentials for agentless connections configured in Cyberwatch, set login/password variable in script
```

Comment:
I think defining these variables here would be more convenient

Reply:
Please explain again how to setup the API keys. The user will not always know the doc.